src/components/Submarine/Submarine.less

```diff
@@ -0,0 +1,215 @@
+@Submarine-Bar-zIndex: 99;
```

Comment:
These should probably fit in with our z-index scale in the variables file. Make sure to test a sidebar in a Dialog!